src/LineaStateBridge.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: MIT
-pragma solidity ^0.8.15;
+pragma solidity ^0.8.19;
```

Comment:
If we specify the concrete single Solidity version in foundry config, we may use "=" instead of "^", if it wouldn't cause any problems. What do you think?

Reply:
we could use 0.8.19 like that it wouldn't give any issue